src/sidebar/store/modules/annotations.ts

```diff
@@ -35,11 +35,6 @@ type AnnotationStub = {
   $tag?: string;
 };
 
-export type UserItem = {
-  user: string;
-  displayName: string | null;
-};
```

Comment:
This type was unused. It was likely something used at some point but then moved to a different module.